src/xenia/app/xenia_main.cc

```diff
@@ -47,7 +47,10 @@
 #include "third_party/xbyak/xbyak/xbyak_util.h"
 
 DEFINE_string(apu, "any", "Audio system. Use: [any, nop, sdl, xaudio2]", "APU");
-DEFINE_string(gpu, "any", "Graphics system. Use: [any, d3d12, vulkan, null]",
+DEFINE_string(gpu, "any",
+              "Graphics system. Use: [any, d3d12, vulkan, null]\nWARNING: "
+              "Vulkan support is obsolete known to be buggy.\nIt is highly "
```

Comment:
"obsolete known", "recommen**e**ded" are not correct in English.

Reply:
I have no idea how I missed that, I have fixed both of the typos in my latest commit.